lib/netext/dialer.go

```diff
@@ -69,17 +69,14 @@ func (d *Dialer) DialContext(ctx context.Context, proto, addr string) (net.Conn,
 	return conn, err
 }
 
-// GetTrail creates a new NetTrail instance with the Dialer
+// Sample creates a new NetTrail instance with the Dialer
```

Comment:
How about naming this something like `ReadsWritesSamples` or `IOSamples`, something more precise? Also, a method comment not longer valid :cry:

Reply:
I meant it more like hte verb "sample" as in dialer samples whatever it has to sample and returns it. But it isn't a thing we usually do so I had it renamed